test-utils/src/main/java/apoc/periodic/PeriodicTestUtils.java

```diff
@@ -41,8 +42,19 @@ public static boolean terminateQuery(String pattern, GraphDatabaseAPI db) {
         return numberOfKilledTransactions > 0;
     }
 
+    // todo - we could get rid of this method (and related code), and leverage the TransactionTestUtil.testTerminateWithCommand
```

Comment:
I'd prefer todos are added as Trello tickets instead of in the code :)

Reply:
I added the trello ticket, id `aWY2zoX5` in `Low priority` column, 
but I can always move it elsewhere, in case :)

Reply:
Awesome, thanks